extract_timeseries_tar.py

```diff
@@ -164,36 +273,64 @@ def get_parser():
             # Note from AB
             # if multiple run, use run 2
             # if multiple session, use ses 1
-            fmri = layout.get(return_type='type', subject=subject, space='MNI152NLin2009cAsym',
+            fmri = layout.get(return_type='type', subject=subject, space=template,
                               desc='preproc', suffix='bold', extension='nii.gz')
+            brain_mask = layout.get(return_type='type', subject=subject, space=template,
+                                    desc='brain', suffix='mask', extension='nii.gz')
+            # TODO: check if brain_mask and fmri always come in pairs
+            # according to doc, desc-preproc_bold and desc-brain_mask should come in pairs
             for ii in range(len(fmri)):
                 file_entitiles = fmri[ii].entities
                 timeseries_root_dir = create_timeseries_root_dir(
                     file_entitiles, output_dir)
                 for dimension in ATLAS_METADATA[atlas_name]['dimensions']:
-                    for resolution in ATLAS_METADATA[atlas_name]['resolutions']:
-                        print(f"\t dim{dimension} - res0{resolution}")
-                        masker, labels = create_atlas_masker(
-                            atlas_name, atlas_path, dimension, resolution)
-                        output_filename = bidsish_timeseries_file_name(
-                            file_entitiles, layout, atlas_name, dimension)
-                        confounds, sample_mask = nilearn.interfaces.fmriprep.load_confounds(fmri[ii].path,
-                                                                                            **LOAD_CONFOUNDS_PARAMS)
-                        timeseries = masker.fit_transform(
-                            fmri[ii].path, confounds=confounds, sample_mask=sample_mask)
-                        # Estimating connectomes
-                        corr_measure = nilearn.connectome.ConnectivityMeasure(
-                            kind="correlation")
-                        connectome = corr_measure.fit_transform([timeseries])[0]
-                        # Save to file
-                        timeseries = pd.DataFrame(timeseries, columns=labels)
-                        timeseries.to_csv(os.path.join(timeseries_root_dir, output_filename), sep='\t', index=False)
-                        connectome = pd.DataFrame(
-                            connectome, columns=labels, index=labels)
-                        connectome.to_csv(
-                            os.path.join(timeseries_root_dir, output_filename.replace("timeseries", "connectome")), sep='\t')
+                    print(f"\tatlas {atlas_name}\tdim{dimension}")
```

Comment:
so you want to fix to just 2mm resolution ?

Reply:
It depends on the input data. One should choose the resolution closest to the one of the processed functional data. 2mm is the finest you can probably get with most fMRI scanning sequence. 
Aside from that, there's no point extracting the same atlas under two different atlas template resolutions.